### DISCUSSION TITLE
Not detecting Huion H420

### DISCUSSION THREAD

**Discussion by RiverBlathe** [OpenTabletDriver/OpenTabletDriver#1484] [Q&A] [answered]
basiclly im just new to the program and am having a hard time getting my tablet to be recognized by the program. it says on the tablets portion of the website that is is supported. just wondering if im missing a step or forgetting to add a pluggin or something. Thanks!

**Comment by EZESJack** [comment]
its not a supported tablet by the drivers, use Huion drivers

**Reply by jamesbt365** [reply]
@EZESJack This is completely false, we do support this tablet. Windows however is different and requires a few steps.

@RiverBlathe if you wanted an answer quicktly then a issue would of been best rather than a discussion but you will need a few things.

Assuming that you are on windows:

First download the pre release here: https://cdn.discordapp.com/attachments/615939905566015493/890625738691211334/OpenTabletDriver.win-x64.zip
once you have that downloaded extract it and make a `userdata` folder beside the rest of the files.

Then download https://github.com/pbatard/libwdi/releases/download/b755/zadig-2.6.exe
Once you have that open sort by all devices then go into the device list and replace the driver on the h420 on interface 0. After you do that get the config from here: https://raw.githubusercontent.com/OpenTabletDriver/OpenTabletDriver/ea6ff0cdf68011d09d884d3fc51dc1eff798cc5b/OpenTabletDriver/Configurations/Huion/H420.json

download that as a .json file and then save it into the `configurations\huion` folder.

**Reply by RiverBlathe** [reply]
Wow thank you. I honestly didn’t think was gonna get an answer. I’ll give that a try. Thanks again!!

**Reply by jamesbt365** [reply]
sorry it took so long, i didn't look at discussions

**Comment by jamesbt365** [comment]
solution listed above in my reply to ezesjack

**Comment by EZESJack** [comment]
i keep getting an open tablet driver daemon fatal error [image: image.png]

**Comment by gonX** [comment, marked as answer]
H420 is now supported in release versions of OpenTabletDriver. If your tablet needs WinUSB (like the H420) please see our wiki entry: https://opentabletdriver.net/Wiki/Install/Windows#winusb
